cloudsync/event.py

```diff
@@ -304,11 +304,13 @@ def _filter_event(self, event: Event, from_walk: bool = False) -> bool:
         if not self._root_path:
             return False
 
-        curr_subpath = self.provider.is_subpath_of_root(event.path or event.oid)
+        curr_subpath = self.provider.is_subpath_of_root(event.path)
         prior_subpath = None
         if self.provider.oid_is_path:
             if not event.prior_oid:
                 # create or delete - ignore if not subpath of root
+                if not event.path:
+                    curr_subpath = self.provider.is_subpath_of_root(event.oid)
```

Comment:
lets just not support this.   if a provider wants filtering ... they need to supply a path... doing it automagically will probably just break things for some provider.s